docker-compose.yaml

```diff
@@ -12,8 +12,6 @@ services:
             - statics:/static
             - solrdata:/solr
             - .:/code
-        ports:
-            - "5000:5000"
```

Comment:
we no longer require to occupy the port 5000 on the machine since we are already exposing it to the nginx container and nginx forwards port 80 to the daisy_web container.